tests/TestUtility/AbstractTestFixture.cs

```diff
@@ -69,6 +71,10 @@ protected Task<OmniSharpWorkspace> CreateWorkspaceAsync(params TestFile[] testFi
 
             var workspace = plugInHost.GetExport<OmniSharpWorkspace>();
 
+            // OmniSharp ships only one provider, CSharpWorkspaceOptionsProvider
+            var formattingProvider = plugInHost.GetExports<IWorkspaceOptionsProvider>().FirstOrDefault() as CSharpWorkspaceOptionsProvider;
+            workspace.Options = formattingProvider?.Process(workspace.Options);
```

Comment:
As written, if the CSharpWorkspaceOptionsProvider isn't included in the MEF composition, this will set workspace.Options to null. Intentional?